apps/prairielearn/src/models/audit-event.ts

```diff
@@ -0,0 +1,29 @@
+import { loadSqlEquiv, queryRows } from '@prairielearn/postgres';
+
+import { type StaffAuditEvent, StaffAuditEventSchema } from '../lib/client/safe-db-types.js';
+
+const sql = loadSqlEquiv(import.meta.url);
+
+export async function selectAuditEvents({
+  subject_user_id,
+  table_name,
+  course_instance_id,
+}: {
+  subject_user_id: string;
+  table_name?: string;
```

Comment:
(drive-by comment, review is still up to Myles) You'll want this to be a mandatory array of table names for two reasons:

- That allows the query planner to utilize the index you added.
- We'll always want to be deliberate about which events we're showing in any given context.

Reply:
Ah, I missed the 'array of table names' piece of this.